flexboxgrid.scss

```diff
@@ -152,20 +148,18 @@ $name: xs;
       width: $container;
     }
 
-    .col-#{$name} {
```

Comment:
@leonardofaria would you mind to explain why you changed `.col-…` to `[class*="col-…"]`?